tests/generic_config_updater/test_lo_interface.py

```diff
@@ -0,0 +1,211 @@
+import logging
+import pytest
+
+from tests.common.helpers.assertions import pytest_assert
+from tests.generic_config_updater.gu_utils import apply_patch, expect_op_success, expect_op_failure
+from tests.generic_config_updater.gu_utils import generate_tmpfile, delete_tmpfile
+from tests.generic_config_updater.gu_utils import create_checkpoint, delete_checkpoint, rollback_or_reload
+
+# Test on t0 topo to verify functionality and to choose predefined variable
+# "LOOPBACK_INTERFACE": {
+    # "Loopback0": {},
+    # "Loopback0|10.1.0.32/32": {},
+    # "Loopback0|FC00:1::32/128": {}
+# }
+pytestmark = [
+    pytest.mark.topology('t0'),
+]
+
+logger = logging.getLogger(__name__)
+
+@pytest.fixture(autouse=True)
+def setup_env(duthosts, rand_one_dut_hostname):
+    """
+    Setup/teardown fixture for each loopback interface test
+    Args:
+        duthosts: list of DUTs.
+        rand_selected_dut: The fixture returns a randomly selected DuT.
+    """
+    duthost = duthosts[rand_one_dut_hostname]
+    create_checkpoint(duthost)
+
+    yield
```

Comment:
what is the purpose of this `yield`?

Reply:
I think this yield is used to split `Setup` from `teardown`. In other words all lines before the `yield` are `Setup` and after are `teardown`